### PR TITLE
Redo d8 1826  individual affinity group tests

### PR DESCRIPTION
**Pull request by jasperlieber**
## Describe context / purpose for this PR
Test individual affinity group pages
(Redo of https://github.com/necyberteam/cyberteam_drupal/pull/818)

Ready for review

## Issue link
https://cyberteamportal.atlassian.net/browse/D8-1826

## Any other related PRs?
https://github.com/necyberteam/cyberteam_drupal/pull/818

## Link to MultiDev instance

http://md-{{ ticket number }}-accessmatch.pantheonsite.io

## Checklist for PR author
- [x] I have checked that the PR is ready to be merged
- [x] I have reviewed the DIFF and checked that the changes are as expected
- [x] I have assigned myself or someone else to review the PR

### CODE REVIEW

File: tests/behat/features/templates/affinity_groups/affinity_groups-individual.feature [hunk 1]
```diff
@@ -1,59 +1,165 @@
...
+    Then I should see "Edit Affinity Group ACCESS Support"
+    # this CI link is created by amp_dev.install
+    When I fill in "Display CI Links on your Affinity Group" with "ci-link-for-user-200"
+    And I wait 1 seconds
+    When I press "Save"
+    And I wait 2 seconds
+    Then I should see "Affinity Group ACCESS Support has been updated."
+    When I follow "ci-link-for-user-200"
+    And I wait 1 seconds
+    Then I should see "ci-link-for-user-200"
```
Comment (jasperlieber) [inline tests/behat/features/templates/affinity_groups/affinity_groups-individual.feature:40]: Note that the above scenario is commented out.  I was able to get this to work locally, but not on github for the reasons described in the comment.

File: tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    # this CI link is created by amp_dev.install
+    When I fill in "Display CI Links on your Affinity Group" with "ci-link-for-user-200"
+    And I wait 1 seconds
+    When I press "Save"
+    And I wait 2 seconds
+    Then I should see "Affinity Group ACCESS Support has been updated."
+    When I follow "ci-link-for-user-200"
+    And I wait 1 seconds
+    Then I should see "ci-link-for-user-200"
+
```
Comment (jasperlieber) [inline tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature:39]: Same as with the templates version of this test, I was not able to add a CI-Link to an AG.

File: tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+  Scenario: Unauthenticated user tests another AG, with a CI dummy_ci_link and a Ci Topic and an Event
+    Given I am not logged in
+    When I am on "/affinity-groups/access-support"
+    Then I should see "Events"
+    Then I should see "[4/04/2023 7:00 PM EDT]"
+    Then link "How to Write a Successful" should contain "/events/6593"
+    # TODO - once able to add a CI Link to this AG, uncomment the following
+    #Then I should see "ci-link-for-user-200"
```
Comment (jasperlieber) [inline tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature:75]: The above should get fixed once a ci-link is added to an AG.

File: tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    When I go to "/affinity-groups/ai-institutes-cyberinfrastructure"
+    Then I should see "NCSA Delta GPU (Delta GPU)"
+    Then I should not see "The Delta GPU resource comprises 4 different node configurations"
+    # TODO -- the UI for ASP for the recommended resources is not the same as the UI for
+    # the cyberteams domains -- the following does not work.  Not sure how to test
+    # these accordion elements.
+    #When I press "NCSA Delta GPU (Delta GPU)"
+    #Then I should see "The Delta GPU resource comprises 4 different node configurations"
+    #When I press "NCSA Delta GPU (Delta GPU)"
+    #Then I should not see "The Delta GPU resource comprises 4 different node configurations"
```
Comment (jasperlieber) [inline tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature:119]: Note this issue also

File: tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
```
Comment (jasperlieber) [inline tests/behat/features/asp/affinity-groups/affinity_groups-individual.feature:138]: same as above

File: tests/behat/features/templates/affinity_groups/affinity_groups-individual.feature [hunk 6]
```diff
@@ -1,59 +1,165 @@
...
+  Scenario: Unauthenticated user tests another AG, with a CI dummy_ci_link and a Ci Topic and an Event
+    Given I am not logged in
+    When I am on "/affinity-groups/access-support"
     Then I should see "Events"
-    Then I should see "Resources"
-    Then I should see "People"
+    Then I should see "[4/04/2023 7:00 PM EDT]"
+    Then link "How to Write a Successful" should contain "/events/6593"
+    # TODO - once able to add a CI Link to this AG, uncomment the following
+    #Then I should see "ci-link-for-user-200"
```
Comment (jasperlieber) [inline tests/behat/features/templates/affinity_groups/affinity_groups-individual.feature:76]: note TODO

**Comment by jasperlieber** [comment]
@a-pasquale & @laheara143  -- this PR passes but with a couple pieces commented out, with notes above.  I'd recommend that we merge this, and return to the TODOs as we have time.  

I'm headed off for a 500 mile bike ride!  See y'all in a week!

**Review by laheara143 — APPROVED** [review]
Looks all set! Will work on TODOS once this is merged

**Pull request merged**